systemtests/src/main/java/io/enmasse/systemtest/GlobalLogCollector.java

```diff
@@ -4,18 +4,31 @@
  */
 package io.enmasse.systemtest;
 
-import io.enmasse.systemtest.cmdclients.KubeCMDClient;
-import io.fabric8.kubernetes.api.model.Pod;
-import org.slf4j.Logger;
-
-import java.io.*;
+import java.io.BufferedReader;
```

Comment:
please clean imports, you don't need to use BufferedReader because you remove it, etc...

Reply:
added commit for this